en/flying/pre_flight_checks.md

```diff
@@ -12,53 +12,64 @@ The sections below list the errors, their likely causes and solutions, and any p
 
 The following errors (with associated checks and parameters) are reported by the [EKF](../advanced_config/tuning_the_ecl_ekf.md) (and propagate to *QGroundControl*):
 
-`PREFLIGHT FAIL: EKF HGT ERROR`:
+#### PREFLIGHT FAIL: EKF HGT ERROR
+
 * This error is produced when the IMU and height measurement data are inconsistent.
 * Perform an accel and gyro calibration and restart the vehicle. If the error persists, check the height sensor data for problems.
 * The check is controlled by the [COM_ARM_EKF_HGT](../advanced_config/parameter_reference.md#COM_ARM_EKF_HGT) parameter.
 
-`PREFLIGHT FAIL: EKF VEL ERROR`:
+#### PREFLIGHT FAIL: EKF VEL ERROR
+
 * This error is produced when the IMU and GPS velocity measurement data are inconsistent. 
 * Check the GPS velocity data for un-realistic data jumps. If GPS quality looks OK, perform an accel and gyro calibration and restart the vehicle.
 * The check is controlled by the [COM_ARM_EKF_VEL](../advanced_config/parameter_reference.md#COM_ARM_EKF_VEL) parameter.
   
-`PREFLIGHT FAIL: EKF HORIZ POS ERROR`:
+#### PREFLIGHT FAIL: EKF HORIZ POS ERROR
+
 * This error is produced when the IMU and position measurement data (either GPS or external vision) are inconsistent. 
 * Check the position sensor data for un-realistic data jumps. If data quality looks OK, perform an accel and gyro calibration and restart the vehicle.
 * The check is controlled by the [COM_ARM_EKF_POS](../advanced_config/parameter_reference.md#COM_ARM_EKF_POS) parameter.
   
-`PREFLIGHT FAIL: EKF YAW ERROR`:
+#### PREFLIGHT FAIL: EKF YAW ERROR
+
 * This error is produced when the yaw angle estimated using gyro data and the yaw angle from the magnetometer or external vision system are inconsistent.
 * Check the IMU data for large yaw rate offsets and check the magnetometer alignment and calibration.
 * The check is controlled by the [COM_ARM_EKF_YAW](../advanced_config/parameter_reference.md#COM_ARM_EKF_YAW) parameter
 * The default value of 0.5 allows the differences between the navigation yaw angle and magnetic yaw angle (magnetometer or external vision) to be no more than 50% of the maximum tolerated by the EKF and provides some margin for error increase when flight commences.
 * It can fail if the yaw gyro has a large offset or if the vehicle is moved or rotated in the presence of a bad magnetic interference or magnetometer calibration.
 
-`PREFLIGHT FAIL: EKF HIGH IMU ACCEL BIAS`:
+#### PREFLIGHT FAIL: EKF HIGH IMU ACCEL BIAS
+
 * This error is produced when the IMU accelerometer bias estimated by the EKF is excessive. 
-* The check is controlled by the [COM_ARM_EKF_AB](../advanced_config/parameter_reference.md#COM_ARM_EKF_AB) parameter.
-  
-`PREFLIGHT FAIL: EKF HIGH IMU GYRO BIAS`:
+* Excessive in this case means that the bias estimate exceeds half the configured limit.
```

Comment:
@bresch Just this and the next item are actual content changes that need to be looked at.

BTW, the code that generates these messages appears to be the call like `mavlink_log_critical(mavlink_log_pub, "Preflight Fail: High Gyro Bias");`. How does that end up being "`PREFLIGHT FAIL: EKF HIGH IMU ACCEL BIAS`" i.e. all caps etc. The reason I ask is that it would be good to grep these and make sure I have them all - but if they don't quite match what users see it is a little tougher.

Reply:
The content is correct. However, it seems that the error messages are a bit outdated... I'll review them.

Reply:
Thanks very much @bresch ! 

@bkueng Eventually is it the intent that these warning get published through the events protocol? I hope so, that would allow us to gather all of the published notifications in one place.

Reply:
Yes. We won't have to document them here then, the description will be directly in code.